bundles/org.openhab.binding.bosesoundtouch/src/main/java/org/openhab/binding/bosesoundtouch/internal/CommandExecutor.java

```diff
@@ -53,6 +57,7 @@ public class CommandExecutor implements AvailableSources {
      * @param handler the handler that created this CommandExecutor
      */
     public CommandExecutor(BoseSoundTouchHandler handler) {
+        mapOfAvailableFunctions = new HashMap<>();
```

Comment:
It's even better to assign the value where the field is declared and add the `final` modifier to the field.